annotationengine/config.py

```diff
@@ -1,25 +1,34 @@
 # Define the application directory
 import os
 from annotationengine.utils import get_app_base_path
+import logging
 
 
 class BaseConfig(object):
+    HOME = os.path.expanduser("~")
     BASE_DIR = os.path.abspath(os.path.dirname(__file__))
     # Statement for enabling the development environment
     DEBUG = True
     proj_dir = os.path.split(get_app_base_path())[0]
     DATASETS = [{
-        'name': 'demo',
-        'CV_SEGMENTATION_PATH': "file://{}/data/segmentation"
+        'name': 'pinky40',
+        'CV_SEGMENTATION_PATH': "gs://neuroglancer/svenmd/pinky40_v11/watershed"
         .format(proj_dir)
     }]
 
     NEUROGLANCER_URL = "https://neuroglancer-demo.appspot.com"
     BIGTABLE_CONFIG = {
         'instance_id': 'pychunkedgraph',
-        'project_id': 'demo'
+        'project_id': "neuromancer-seung-import"
     }
-
+    TESTING = False
+    LOGGING_FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
+    LOGGING_LOCATION = HOME + '/annoeng/bookshelf.log'
+    LOGGING_LEVEL = logging.DEBUG
+    # TODO what is this suppose to be by default?
+    CHUNKGRAPH_TABLE_ID = "chunkedgraph"
+
+    CHUNKGRAPH_TABLE_ID = "pinky40_fanout2_v7"
```

Comment:
I think this will work for now, but really we need to have dataset specific endpoints on the pychunkedgraph if we are going to deploy these together.